docs/index.rst

```diff
@@ -6,6 +6,15 @@
 FESOM2 documentation
 ====================
 
+The Finite Element Sea Ice-Ocean Model (FESOM2).
```

Comment:
The FESOM 2 is finite-volume version, so: "Finite volumE Sea Ice-Ocean Model (FESOM2)"

Reply:
Done!